app/controllers/schools/placement_requests/acceptance/preview_confirmation_email_controller.rb

```diff
@@ -16,7 +16,7 @@ def update
           @booking = @placement_request.booking
           @booking.assign_attributes(booking_params)
 
-          if @booking.valid?(:acceptance_email_preview) && @booking.accept! && candidate_booking_notification(@booking).despatch_later!
+          if @booking.valid?(:acceptance_email_preview) && @booking.accept! && candidate_booking_notifications(@booking)
```

Comment:
I find this a bit unintuitive to have something in the conditional being an action that sends an SMS; is it worth calling out separately?